lib/please_cli/please_cli/decision_task.py

```diff
@@ -511,6 +511,13 @@ def cmd(ctx,
         deploy_tasks = {}
         for index, (project, project_requires, deploy_target, deploy_options) in \
                 enumerate(sorted(projects_to_deploy, key=lambda x: x[0])):
+            try:
+                enable = deploy_options['enable']
+            except KeyError:
+                raise click.ClickException('Missing `enable` in deploy options' )
```

Comment:
:+1: can you also mention which project and channel this is enable option is missing at?